contrib/docker/docker-compose.yml

```diff
@@ -15,10 +15,10 @@ services:
     restart: unless-stopped
     # See the readme for a full documentation of the environment settings
     environment:
-      - SYNAPSE_CONFIG_PATH=/etc/homeserver.yaml
+      - SYNAPSE_CONFIG_PATH=/data/homeserver.yaml
```

Comment:
surely this is inconsistent with your mapping to /etc?

Reply:
Oh, you are right. Going to fix that.